src/components/electricity_tariffs/energycharts/electricity_tariff.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="electricity-tariff-energycharts">
+		<openwb-base-select-input
+			title="Land"
+			notSelected="Bitte auswählen"
+			:options="[
+				{ value: 'DE-LU', text: 'Deutschland' },
+				{ value: 'AT', text: 'Österreich' },
+				{ value: 'CH', text: 'Schweiz' },
+			]"
+			:model-value="electricityTariff.configuration.country"
+			@update:model-value="
+				updateConfiguration($event, 'configuration.country')
+			"
+		>
+			<template #help>
+				Börsenstrompreise von Energy Charts (energy-charts.info) 
+				Die Rohdaten werden von Wissenschaftlern des Fraunhofer-Institut 
+				für Solare Energiesysteme ISE aus zahlreichen Quellen stündlich 
+				oder täglich abgerufen und für die Darstellung aufbereitet.
+ 			</template>
+		</openwb-base-select-input>
```

Comment:
Der Hilfetext gehört nicht zu der Einstellung "Land", sondern ist eine allgemeine Info zu dem Modul. Bitte in einen eigenen Info-Block direkt nach `<div class="electricity-tariff-energycharts">` verschieben.
```suggestion
		<openwb-base-alert subtype="info">
			Börsenstrompreise von Energy Charts (energy-charts.info)<br />
			Die Rohdaten werden von Wissenschaftlern des Fraunhofer-Institut
			für Solare Energiesysteme ISE aus zahlreichen Quellen stündlich
			oder täglich abgerufen und für die Darstellung aufbereitet.
		</openwb-base-alert>
		<openwb-base-select-input
			title="Land"
			notSelected="Bitte auswählen"
			:options="[
				{ value: 'DE-LU', text: 'Deutschland' },
				{ value: 'AT', text: 'Österreich' },
				{ value: 'CH', text: 'Schweiz' },
			]"
			:model-value="electricityTariff.configuration.country"
			@update:model-value="
				updateConfiguration($event, 'configuration.country')
			"
		/>
```